sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/TypesSamples.cs

```diff
@@ -0,0 +1,140 @@
+            var variableName = "aggregateVariable";
+            variables.Add(variableName, aggregateVariable);
+
+            var timeSeriesTypesProperties = new Dictionary<string, string>
+            {
+                { "Type1", Guid.NewGuid().ToString()},
+                { "Type2", Guid.NewGuid().ToString()}
+            };
+
+            foreach (KeyValuePair<string, string> property in timeSeriesTypesProperties)
+            {
+                var type = new TimeSeriesType(property.Key, variables)
+                {
+                    Id = property.Value
+                };
+                timeSeriesTypes.Add(type);
+            }
+
+            Response<TimeSeriesTypeOperationResult[]> createTypesResult = await client
+                .Types
+                .CreateOrReplaceAsync(timeSeriesTypes)
+                .ConfigureAwait(false);
+
+            // Check if the result error array does contain any object that is set
+            foreach (var result in createTypesResult.Value)
+            {
+                if (result.Error != null)
+                {
+                    Console.WriteLine($"Failed to create a Time Series Insights type: {result.Error.Message}.");
+                }
+            }
+            #endregion Snippet:TimeSeriesInsightsSampleCreateType
+
+            #region Snippet:TimeSeriesInsightsSampleGetTypeById
+            // Code snippet below shows getting a default Type using Id
+            Response<TimeSeriesTypeOperationResult[]> getTypeByIdResults = await client
+                .Types
+                .GetByIdAsync(new string[] { modelSettings.DefaultTypeId })
+                .ConfigureAwait(false);
+
+            TimeSeriesTypeOperationResult getTypeByIdResult = getTypeByIdResults.Value[0];
+
+            if (getTypeByIdResult.TimeSeriesType != null)
+            {
+                Console.WriteLine($"Retrieved Time Series Insights type with Id '{getTypeByIdResult.TimeSeriesType.Id}' and name '{getTypeByIdResult.TimeSeriesType.Name}'.");
+            }
+            else if (getTypeByIdResult.Error != null)
+            {
+                Console.WriteLine($"Failed to retrieve a Time Series Insights type with Id '{getTypeByIdResult.Error.Message}'.");
+            }
+            #endregion Snippet:TimeSeriesInsightsSampleGetTypeById
+
+            #region Snippet:TimeSeriesInsightsSampleReplaceType
+            // Update variables with adding a new variable
+            foreach (TimeSeriesType type in timeSeriesTypes)
+            {
+                type.Description = "Description";
+            }
+
+            Response<TimeSeriesTypeOperationResult[]> updateTypesResult = await client
+                .Types
+                .CreateOrReplaceAsync(timeSeriesTypes)
+                .ConfigureAwait(false);
+
+            // Check if the result error array does contain any object that is set
+            foreach (var result in createTypesResult.Value)
```

Comment:
nit: maybe a comment similar to this:
// The response of calling the API contains a list of error objects corresponding by position to the input parameter
            // array in the request. If the error object is set to null, this means the operation was a success.